doc/source/examples/metrology/Creating Network Definitions for SOLT Calibration Standards.ipynb

```diff
@@ -0,0 +1,1116 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 12,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def rs_to_keysight(rs_offset_length, rs_offset_loss, offset_z0=50):\n",
+    "    offset_delay = rs_offset_length / skrf.constants.c\n",
+    "    offset_loss = skrf.mathFunctions.db_2_np(rs_offset_loss * offset_z0 / offset_delay)\n",
+    "    return offset_delay, offset_loss"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "After unit conversion, we can define standards just like how calibration standards in Keysight-style\n",
+    "coefficients are defined."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 13,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "offset_delay, offset_loss = rs_to_keysight(4.344e-3, 0.0033)\n",
+    "gamma_l, zc = offset_gamma_and_zc(offset_delay, offset_loss)\n",
+    "medium_open = DefinedGammaZ0(\n",
+    "    frequency=freq,\n",
+    "    gamma=gamma_l, Z0=zc, z0=50\n",
+    ")\n",
+    "line_open = medium_open.line(\n",
+    "    d=1, unit='m',\n",
+    "    z0=medium_open.Z0, embed=True\n",
+    ")\n",
+    "\n",
+    "offset_delay, offset_loss = rs_to_keysight(5.0017e-3, 0.0038)\n",
+    "gamma_l, zc = offset_gamma_and_zc(offset_delay, offset_loss)\n",
+    "medium_short = DefinedGammaZ0(\n",
+    "    frequency=freq,\n",
+    "    gamma=gamma_l, Z0=zc, z0=50\n",
+    ")\n",
+    "line_short = medium_short.line(\n",
+    "    d=1, unit='m',\n",
+    "    z0=medium_short.Z0, embed=True\n",
+    ")\n",
+    "\n",
+    "offset_delay, offset_loss = rs_to_keysight(17.375e-3, 0.0065)\n",
+    "gamma_l, zc = offset_gamma_and_zc(offset_delay, offset_loss)\n",
+    "medium_thru = DefinedGammaZ0(\n",
+    "    frequency=freq,\n",
+    "    gamma=gamma_l, Z0=zc, z0=50\n",
+    ")\n",
+    "line_thru = medium_thru.line(\n",
+    "    d=1, unit='m',\n",
+    "    z0=medium_thru.Z0, embed=True\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Modeling the Shunt Impedance\n",
+    "\n",
+    "The definition of shunt impedance is identical to the Keysight format.\n",
+    "\n",
+    "But, beware of the units used for the capacitance and inductance! In the\n",
+    "given table, the capacitances are given in $10^{−15} \\text{ F}$, $10^{−15} \\text{ F/Hz}$,\n",
+    "$10^{−15} \\text{ F/Hz}^2$, and\n",
+    "$10^{−15} \\text{ F/Hz}^3$. For Keysight and Anritsu VNAs, they're given in $10^{-15} \\text{ F}$,\n",
+    "$10^{-27} \\text{ F/Hz}$, $10^{-36} \\text{ F/Hz}^2$ and $10^{-45} \\text{ F/Hz}^3$. Inductance\n",
+    "units have the same differences. Always double-check the units before start modeling. To\n",
+    "convert the units from the first to the second format, multiply $x_1$, $x_2$ and $x_3$\n",
+    "by 1000 (don't change the constant term $x_0$). For consistency, we'll use the second format\n",
```

Comment:
This doesn't calculate correctly. I think the units should be F/GHz? The table also has the same 1e-15*F/Hz units which are way too high when compared to the Keysight coefficients. F/GHz units should be consistent with the current calculations.

Reply:
You're right, the unit is per GHz.